flecsi/flog.hh

```diff
@@ -31,7 +31,9 @@
 #include <unordered_set>
 #include <vector>
 
+#if !defined(_WIN32)
 #include <unistd.h>
+#endif
```

Comment:
Do we still need this at all if we use `sleep_for`?

Reply:
Not sure what `unistd.h` was #included for in the first place. But yes, excellent point - we might be able to avoid having it altogether.